components/org.wso2.carbon.identity.sso.agent/src/main/java/org/wso2/carbon/identity/sso/agent/util/SSOAgentUtils.java

```diff
@@ -368,4 +442,16 @@ private static Document getDocument(DocumentBuilderFactory documentBuilderFactor
         ByteArrayInputStream inputStream = new ByteArrayInputStream(samlString.getBytes());
         return docBuilder.parse(inputStream);
     }
+
+    public static boolean isArtifactResolveReqSigningEnabled(Map<String, String> properties) {
+
+        if (properties != null) {
+            String prop =
+                    properties.get(SSOAgentConstants.SSOAgentConfig.SAML2.IS_ARTIFACT_RESOLVE_REQ_SIGNED);
+            if (StringUtils.isNotBlank((prop))) {
```

Comment:
Do we really need this check ?. Isn't Boolean. parseBoolean handle empty values ?